app/helpers/dashboard/sub_service_requests_helper.rb

```diff
@@ -263,4 +263,21 @@ def ssr_admin_button(ssr, user, permission_to_edit, admin_access)
       ''
     end
   end
+
+  def ssr_select_options(ssr)
+    ssr.nil? ? [] : statuses_with_classes(ssr)
+  end
+
+  private
+
+  def statuses_with_classes(ssr)
+    ssr.organization.get_available_statuses.invert.map do |status|
+      if status.include?('Complete') || status.include?('Withdrawn')
```

Comment:
I think options_for_select takes an array of arrays. That's what this method is returning.